python/lsst/rucio/register/data_type.py

```diff
@@ -0,0 +1,25 @@
+# This file is part of rucio_register
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+class DataType:
+    DATA_PRODUCT = "data_product"
+    RAW_FILE = "raw_file"
```

Comment:
It's just a suggestion that is feel free to leave things as they are, but as you're using Python 3.11 you may consider [enum.StrEnum]() when creating a specific set of symbolic constants with unique values. An enum offers a better type safety. For example, making `rubin_butler_type` an enum would ensure that only expected values are passed in its place when calling  `getRucioInterface()`.

Reply:
Agreed, but I'm going to leave this for now but will change it for the next pending ticket I have for this.